WordPress/gutenberg
# Migrate Cover Block tests to Playwright

## What?

This PR is part of the migration effort tracked at https://github.com/WordPress/gutenberg/issues/38851.

## Why?

This PR proposes a migration of the `cover.test.js` to the new Playwright-based framework.

## How?


## Testing Instructions




`npm run test:e2e:playwright -- test/e2e/specs/editor/blocks/cover.spec.js`

In the current form (pending outcome of [this question](https://github.com/WordPress/gutenberg/pull/45784#issuecomment-1523034256), it should produce the following outcome:

```
Running 6 tests using 1 worker

  ✓  1 …2 › Cover › can set overlay color using color picker on block placeholder (2.4s)
Failed to load resource: the server responded with a status of 404 (Not Found)
  ✓  2 … Cover › can set background image using image upload on block placeholder (3.3s)
Failed to load resource: the server responded with a status of 404 (Not Found)
  ✓  3 …over.spec.js:102:2 › Cover › dims background image down by 50% by default (1.8s)
Failed to load resource: the server responded with a status of 404 (Not Found)
  ✓  4 …] › editor/blocks/cover.spec.js:126:2 › Cover › can have the title edited (2.3s)
Failed to load resource: the server responded with a status of 404 (Not Found)
  ✓  5 …tor/blocks/cover.spec.js:151:2 › Cover › can be resized using drag & drop (2.4s)
Failed to load resource: the server responded with a status of 404 (Not Found)
  -  6 …er › dims the background image down by 50% when transformed from the Image block

  1 skipped
  5 passed (13.1s)
```

## Screenshots or screencast
